service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentSealingServiceTest.java

```diff
@@ -84,4 +87,21 @@ void shouldSealAndUploadDocumentWithoutFileConversion() throws Exception {
         assertThat(actualSealedDocumentReference).isEqualTo(sealedDocumentReference);
         assertThat(actualDocumentBinaries.getValue()).isEqualTo(expectedSealedDocumentBinaries);
     }
+
+    @Test
+    void shouldThrowExceptionWhenDocumentIsNotPdf() {
+        final String fileName = "test.pdf";
+        final byte[] notPdf = new byte[]{1};
+        final byte[] inputDocumentBinaries = readBytes("documents/document.pdf");
+        final DocumentReference inputDocumentReference = testDocumentReference(fileName);
+
+        when(documentDownloadService.downloadDocument(inputDocumentReference.getBinaryUrl()))
+            .thenReturn(inputDocumentBinaries);
+        when(documentConversionService.convertToPdf(inputDocumentBinaries, fileName)).thenReturn(notPdf);
+
+        final Exception exception = assertThrows(Exception.class,
+            () -> documentSealingService.sealDocument(inputDocumentReference));
+
+        assertThat(exception).isInstanceOf(UncheckedIOException.class);
```

Comment:
assertThrows(UncheckedIOException.class,
            () -> documentSealingService.sealDocument(inputDocumentReference));

should be enough